src/main/webapp/courseLoader.js

```diff
@@ -0,0 +1,124 @@
+    hourChart.draw(hourData, hourOptions);
+
+
+    const tempDiffList = dataObject.hoursList;
+    const diffList = [["difficulty"],[1],[4]].concat(tempDiffList);
+    const diffData = new google.visualization.arrayToDataTable(diffList)
+    const diffOptions = {
+        title: "Difficulty of Class",
+        legend: {position: "none"},
+        vAxis: {title: "# Students"},
+        hAxis: {title: "Difficulty"},
+        histogram: {
+        hideBucketItems: true
+        }
+      };
+    const diffChart = new google.visualization.Histogram(document.getElementById("diff-chart"));
+    diffChart.draw(diffData, diffOptions);
+
+
+    const tempTermPerceptionList = dataObject.termPerceptionList;
+    const termPerceptionList = [["Term Perception"],[11],[5]].concat(tempTermPerceptionList);
+    const termPerceptionData = new google.visualization.arrayToDataTable(termPerceptionList);
+    const termPerceptionOptions = {
+        title: "Perception of Term Reviews",
+        legend: {position: "none"},
+        vAxis: {title: "Perception"},
+        hAxis: {title: "Comment Quantity"},
+        histogram: {
+        hideBucketItems: true
+        }
+      };
+    const termPerceptionChart = new google.visualization.Histogram(document.getElementById("term-chart"));
+    termPerceptionChart.draw(termPerceptionData, termPerceptionOptions);
+
+
+    const tempProfPerceptionList = dataObject.termPerceptionList;
+    const profPerceptionList = [["Professor Perception"],[2],[9]].concat(tempProfPerceptionList);
+    const profPerceptionData = new google.visualization.arrayToDataTable(profPerceptionList);
+    const profPerceptionOptions = {
+        title: "Perception of Professor Reviews",
+        legend: {position: "none"},
+        vAxis: {title: "Perception"},
+        hAxis: {title: "Comment Quantity"},
+        histogram: {
+        hideBucketItems: true
+        }
+      };
+    const profPerceptionChart = new google.visualization.Histogram(document.getElementById("prof-chart"));
+    profPerceptionChart.draw(profPerceptionData, profPerceptionOptions);
+}
+
+function passData(){
+    const queryString = window.location.search;
+    const urlParams = new URLSearchParams(queryString);
+    const courseName = urlParams.get("course-name");
+    const term = urlParams.get("term");
+    const profName = urlParams.get("prof-name");
+    const units = urlParams.get("num-units")
+    const schoolName = urlParams.get("school-name")
+
+    const termInput = document.getElementsByName("term-input")[0].value;
+    const profInput = document.getElementsByName("prof-input")[0].value;
+    const ratingTerm = document.getElementById("rating-term").value;
+    const ratingProf = document.getElementById("rating-prof").value;
+    const hours = document.getElementById("hoursOfWork").value;
+    const diff = document.getElementById("difficulty").value;
+
+    const url = new URL("/data", window.location.origin);
+    url.searchParams.set("course-name", courseName);
```

Comment:
Maybe add a TODO here as a reminder to switch to putting these params into the request body rather than the path.